File: PLD1_InputProcessOutput/EX11_cookieCalories/cookieCalories.cpp
#include <iostream>
using namespace std;

int main()
{
    int cookies;
    int calories = 300;
    int totalCalories;

    // Input
    cout << "Enter a cookies ate : ";
    cin >> cookies;

    // Process
    totalCalories = cookies * calories;

    // Output
    cout << "Total Calories consumed :" << totalCalories << endl;

    return 0;
}

File: CH1-input-process-output/EX11-cookie-calories/cookieCalories.java
import java.util.Scanner;

public class cookieCalories {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int cookies;
        int calories = 300;
        int totalCalories;

        // Input
        System.out.print("Enter a cookies ate : ");
        cookies = input.nextInt();

        // Process
        totalCalories = cookies * calories;

        // Output
        System.out.print("Total Calories Consumed : " + totalCalories);
        input.close();
    }
}